model/lifecycle.go

```diff
@@ -768,10 +772,14 @@ func createTasksForBuild(project *Project, buildVariant *BuildVariant, b *build.
 		if err != nil {
 			return tasks, errors.Wrapf(err, "Failed to create display task %s", id)
 		}
+		dependencyIds := map[string]bool{}
```

Comment:
I feel like I'm missing something, but isn't t set by the call to createDisplayTask above, which doesn't set the dependencies? So I don't think this block does anything

Reply:
okay whoa true